string model_name                 # name of the model to be spawn
string model_xml                  # this should be an urdf or gazebo xml
geometry_msgs/Pose initial_pose   # only applied to canonical body
string reference_frame            # initial_pose is defined relative to the frame of this model/body
